cmd/project_create.go

```diff
@@ -66,6 +67,7 @@ lab project create -n "new proj"           # user/new-proj named "new proj"`,
 		if err != nil {
 			log.Fatal(err)
 		}
+		spew.Dump(p)
```

Comment:
@zaquestion did you mean to leave this in? My Go is rusty, but this looks like a debugging statement.